lib/src/arithmetic/values/floating_point_values/floating_point_value_populator.dart

```diff
@@ -56,6 +59,7 @@ class FloatingPointValuePopulator<FpvType extends FloatingPointValue> {
     required LogicValue sign,
     required LogicValue exponent,
     required LogicValue mantissa,
+    bool explicitJBit = false,
```

Comment:
this `explicitJBit` argument is not connected to anything.  did you mean to connect it to the `_unpopulated`?

also, is is a sign of some missing testing somewhere that nothing caught this?

Reply:
uh oh.  This is strange because I was fixing bugs that varied based on explicit vs not.

Reply:
This argument is not needed.  The FP itself contains the _explicitJBit that it uses to setup unpopulated.